### ISSUE TITLE
Indicate BlueJean room is in use

### ISSUE THREAD

**Issue by jlost** [tl-its-umich-edu/remote-office-hours-queue#22]
Add an indication in the UI of the BlueJeans participants presence.

Feedback from an advisor in support of this issue:

> I was wondering if there is some way for the system to be able to recognize if a student actually has their blue jeans link open. This would be helpful because sometimes a student is next in line but they are not in the blue jeans meeting for whatever reason. I can go to the next person in line but I have no idea who has their chat window open so I end up clicking on each person until I eventually get to someone in the chat. Also, I end up having to go back to the people who I skipped over because their video wasn't open but I have no idea if the situation has changed to more often than not I just end up clicking the meeting link every few minutes in hopes that maybe the student has finally opened their BlueJeans meeting.

-- @mfldavidson

**Comment by jlost** [comment]
I spent some time looking at this recently and I think the best we can do is use the Bluejeans `live_meeting` endpoint to determine whether the meeting is in progress or not. We discussed the performance impact a little bit in slack but I think there would need to be some more discussion there if we decide to go that route.

James also mentioned just tracking when the "Join Meeting" button is clicked, which is another option.

-- @psteffes

**Comment by teshsopher** [comment]
I've mocked up this design here: https://www.sketch.com/s/d3e8e90e-86a6-4d57-ba0e-33e40954eda9

As a note, the designs still need more testing/feedback. @mfldavidson , do you have any current/potential hosts that would be willing to test these with me?

**Comment by mfldavidson** [comment]
Absolutely! I should probably just ask people formally if they are willing to be beta testers and make an email group so you can contact them. I'll go ahead and ask a few folks if they'd be ok with me doing that and get back to you ASAP!

**Comment by jlost** [comment]
Blocked by #39

**Comment by mfldavidson** [comment]
With Zoom this becomes https://marketplace.zoom.us/docs/api-reference/webhook-reference/meeting-events/meeting-participant-waiting-for-host
